### DISCUSSION TITLE
Create a Grid in a dialog form

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#7087] [Help]
How can I instantiate a grid in a dialog form that will list details of the dialog form
Note that this is not a **MasterDetailRelation**, I just want to display in a grid the data that is saved from the form.
I want to create the grid in the red circled area and fill the grid anytime data is saved from the form

![image](https://github.com/serenity-is/Serenity/assets/50826372/07e43974-cd7e-4185-8975-7618b2ffa4e5)

**Comment by LSNicholls** [comment]
Hi @rhemasystems - how about this:

1. create a view on the entity
2. create a new entity from the view
3. relate the new view-entity (which presumably is going to be readonly, it's just for display) to the original entity
4. treat the new view-entity as if it is in a master-child relationship, for the purpose of the grid that you want.

This is a kind of interesting question, btw, so I'd love to hear more about your requirement for this.  What does the grid provide from a user experience point of view?  Not arguing, just trying to learn.

**Reply by rhemasystems** [reply]
Thank you @LSNicholls,
The grid will just list all the entries that have been saved by the user after clicking the Save button.

**Reply by LSNicholls** [reply]
@rhemasystems,  
> all the entries

I'm not clear on if you mean "all the columns that they changed", maybe pivoted to multiple grid rows, or "all the rows that they changed", maybe meaning a history of this row, or related entities?

Regardless, I'd still create a view for this and I think it would work.
